### DISCUSSION TITLE
Is there a way to activate global auto-type from the CLI?

### DISCUSSION THREAD

**Discussion by paddylandau** [keepassxreboot/keepassxc#6858] [Q&A] [answered]
I have a specific use case where it would be greatly helpful if I could activate the global auto-type using a command instead of the usual keyboard shortcut.

Is this possible? Something like, say, `keepassxc autotype`?

I've looked at `keepassxc.cli`, and I don't see a suitable command.

In case it makes a difference, I have KeePassXC v2.6.6 running on Linux Ubuntu 20.04.

Thank you

**Comment by droidmonkey** [comment, marked as answer]
No this is not possible

**Reply by paddylandau** [reply]
Thanks for the answer.

I managed to figure out a way: `xdotool` does the trick, although it needs some fiddling to ensure that it works.

**Reply by paddylandau** [reply]
@droidmonkey — I'd like to mark your comment as an answer, but github isn't allowing me to do so. The "Mark as answer" button is greyed out, and a comment telling me to mark the comment as an answer is obscuring a part of the screen. So, if you can mark your comment as an answer, please do so!
